lib/rules/sort-comp.js

```diff
@@ -131,84 +131,108 @@ module.exports = {
      * @returns {Array} The matching patterns indexes. Return [Infinity] if there is no match.
      */
     function getRefPropIndexes(method) {
-      let isRegExp;
-      let matching;
-      let i;
-      let j;
-      const indexes = [];
+      const methodGroupIndexes = [];
 
-      if (method.getter) {
-        const getterIndex = methodsOrder.indexOf('getters');
-        if (getterIndex >= 0) {
-          indexes.push(getterIndex);
-        }
-      }
-
-      if (method.setter) {
-        const setterIndex = methodsOrder.indexOf('setters');
-        if (setterIndex >= 0) {
-          indexes.push(setterIndex);
-        }
-      }
+      for (let groupIndex = 0; groupIndex < methodsOrder.length; groupIndex++) {
```

Comment:
could this be a `.map` instead of a for loop?

Reply:
I will update this to be a `forEach` (since we don't care about the resulting array that a `map` creates).

I had considered making it a `forEach`, but reading the code I had assumed that `for`s were preferable since there's a lot of places that `forEach` could be used but is not.